src/scale.ts

```diff
@@ -355,6 +359,7 @@ export function channelScalePropertyIncompatability(channel: Channel, propName:
     case 'exponent':
     case 'nice':
     case 'zero':
+    case 'interpolate':
```

Comment:
Have you thought carefully if interpolate works with any encoding channel?  Unless I overlook anything, it's only for color.  In any case, please ask rather than slipping in random code if you're not sure -- otherwise you are adding a new bug to the system.  

If you agree that it's only for color, move it next to scheme and consolidate the error message.